rosdep/python.yaml

```
@@ -6008,6 +6008,13 @@ python3-bitarray:
   gentoo: [dev-python/bitarray]
   nixos: [python3Packages.bitarray]
   ubuntu: [python3-bitarray]
+python3-bitstring:
+  debian: [python3-bitstring]
+  fedora: [python3-bitstring]
+  gentoo: [dev-python/bitstring]
+  opensuse: [python3-bitstring]
+  rhel: [python3-bitstring]
```

```suggestion
  rhel:
    '*': [python3-bitstring]
    '7': [python36-bitstring]
```


In EPEL 7 I see `python36-bitstring` but I don't see `python-bitstring` https://dl.fedoraproject.org/pub/epel/7/x86_64/Packages/p/

---

Good catch! I missed that searching for Fedora. I tired to find this with no luck unless it is the exact string `python36-bitstring`. For example https://packages.fedoraproject.org/search?query=*bitstring only shows the python2 version :confused: